pkg/apis/policies/v1/policyserver_webhook.go

```diff
@@ -50,3 +63,44 @@ func (ps *PolicyServer) Default() {
 		controllerutil.AddFinalizer(ps, constants.KubewardenFinalizer)
 	}
 }
+
+// +kubebuilder:webhook:path=/validate-policies-kubewarden-io-v1-policyserver,mutating=false,failurePolicy=fail,sideEffects=None,groups=policies.kubewarden.io,resources=policyservers,verbs=create;update,versions=v1,name=vpolicyserver.kb.io,admissionReviewVersions=v1
+
+// polyServerValidator validates PolicyServers
+type policyServerValidator struct {
+	k8sClient            client.Client
+	deploymentsNamespace string
+}
+
+func (v *policyServerValidator) validate(ctx context.Context, obj runtime.Object) error {
+	policyServer, ok := obj.(*PolicyServer)
+	if !ok {
+		return fmt.Errorf("expected a PolicyServer object, got %T", obj)
+	}
+
+	// The PolicyServer name must be maximum 63 like all Kubernetes objects to fit in a DNS subdomain name
+	if len(policyServer.GetName()) > validationutils.DNS1035LabelMaxLength {
+		return fmt.Errorf("spec.ImagePullSecret cannot be longer than %d characters", validationutils.DNS1035LabelMaxLength)
```

Comment:
I think the error message is wrong, it's the policy-server name that is longer than 64 chars, not the ImagePullSecret

Reply:
thanks!